backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -142,79 +121,43 @@ private async void OnIsarRobotInfo(object? sender, MqttReceivedArgs mqttArgs)
 
                 UpdatePortIfChanged(isarRobotInfo.Port, ref robot, ref updatedFields);
 
-                if (isarRobotInfo.CurrentInstallation is not null)
-                {
-                    UpdateCurrentInstallationIfChanged(isarRobotInfo.CurrentInstallation, ref robot, ref updatedFields);
-                }
+                if (isarRobotInfo.CurrentInstallation is not null) { UpdateCurrentInstallationIfChanged(isarRobotInfo.CurrentInstallation, ref robot, ref updatedFields); }
+                if (updatedFields.IsNullOrEmpty()) { return; }
 
-                if (!updatedFields.IsNullOrEmpty())
-                {
-                    robot = await robotService.Update(robot);
-                    _logger.LogInformation(
-                        "Updated robot '{id}' ('{robotName}') in database: {updates}",
-                        robot.Id,
-                        robot.Name,
-                        updatedFields
-                    );
-                }
+                robot = await robotService.Update(robot);
+                _logger.LogInformation("Updated robot '{Id}' ('{RobotName}') in database: {Updates}", robot.Id, robot.Name, updatedFields);
 
             }
-            catch (DbUpdateException e)
-            {
-                _logger.LogError(e, "Could not add robot to db");
-            }
-            catch (Exception e)
-            {
-                _logger.LogError(e, "Could not update robot in db");
-            }
+            catch (DbUpdateException e) { _logger.LogError(e, "Could not add robot to db"); }
+            catch (Exception e) { _logger.LogError(e, "Could not update robot in db"); }
         }
 
-        private static void UpdateVideoStreamsIfChanged(List<CreateVideoStreamQuery> videoStreamQueries,
-            ref Robot robot, ref List<string> updatedFields)
+        private static void UpdateVideoStreamsIfChanged(List<CreateVideoStreamQuery> videoStreamQueries, ref Robot robot, ref List<string> updatedFields)
         {
             var updatedStreams = videoStreamQueries
                 .Select(
                     stream =>
                         new VideoStream
                         {
-                            Name = stream.Name,
-                            Url = stream.Url,
-                            Type = stream.Type
+                            Name = stream.Name, Url = stream.Url, Type = stream.Type
                         }
                 )
                 .ToList();
 
             var existingVideoStreams = robot.VideoStreams;
-            if (
-                updatedStreams.Count == robot.VideoStreams.Count
-                && updatedStreams.TrueForAll(stream => existingVideoStreams.Contains(stream))
-            )
-            {
-                return;
-            }
+            if (updatedStreams.Count == robot.VideoStreams.Count && updatedStreams.TrueForAll(stream => existingVideoStreams.Contains(stream))) { return; }
```

Comment:
And here, etc (there are more but I don’t want to spam you. Also this is subjective so if you prefer with the braces then that’s fine)